listings/ch00-introduction/if_else/src/access_control.cairo

```diff
@@ -1,23 +1,30 @@
-use starknet::ContractAddress;
-
 #[contract]
 mod SimpleAccessControl {
-    use starknet::get_caller_address;
+    use starknet::{get_caller_address,ContractAddress};
 
     struct Storage {
-        owner: starknet::ContractAddress, 
+        owner: ContractAddress, 
     }
 
+    #[event]
+    fn welcomeEvent(name: felt252) {}
+
     #[constructor]
-    fn constructor(_address: starknet::ContractAddress) {
+    fn constructor(_address: ContractAddress) {
```

Comment:
Just one last remark on convention: For now, the common pattern is to have parameters without `_` prefix, and storage variables with a `_` prefix.

Also, events are written in `PascalCase`